vars/piperExecuteBin.groovy

```diff
@@ -78,6 +78,9 @@ void call(Map parameters = [:], String stepName, String metadataFile, List crede
                         try {
                             try {
                                 credentialWrapper(config, credentialInfo) {
+                                    echo "[DEBUG] Current environment limits:"
```

Comment:
can we enable this output if verbose: true set?

Reply:
if it's cloudFoundry specific, then I'd rather remove this debug logs

Reply:
> if it's cloudFoundry specific, then I'd rather remove this debug logs from piperExecuteBin part (and possible place it to cloudFoundry step, if this is realy needed)

I suppose this log output could be useful for other steps that use dockerExecute environment. Proposed log is a shell command, not Go code. It served to measure current execution environment limits before running actual Piper bin.

piperExecuteBin is a wrapper step called by other steps which wants to be executed inside docker/kubernetes environment. So this debug log executes inside container (exactly what we need) just before invoking Piper CLI with related step.

In simple words, the call chain looks like below (on the example of CloudFoundryDeploy, but we have >100 steps that work the same):
1. Orchestrator (Jenkins) first invokes related step Groovy code.
2. Then that Groovy [step invokes](https://github.com/SAP/jenkins-library/blob/93c4ea61aa8393fe5467a4cc9dc412f9a7b802bf/vars/cloudFoundryDeploy.groovy#L27) piperExecuteBin.
3. Then piperExecuteBin invokes [related step Go code](https://github.com/SAP/jenkins-library/blob/93c4ea61aa8393fe5467a4cc9dc412f9a7b802bf/vars/piperExecuteBin.groovy#L73-L81) (inside container via dockerExecute wrapper step [because of dockerWrapper](https://github.com/SAP/jenkins-library/blob/93c4ea61aa8393fe5467a4cc9dc412f9a7b802bf/vars/piperExecuteBin.groovy#L161-L168)).

Reply:
> can we enable this output if verbose: true set?

Makes sense. I updated PR.